### ISSUE TITLE
Add highlighting for Kotlin

### ISSUE THREAD

**Issue by daincredibleholg** [Automattic/syntaxhighlighter#227]
### Is your feature request related to a problem? Please describe
I am one of the Java Advent organizers and with Kotlin's popularity going further through the roof it starts getting problematic that articles about Kotlin can't have properly highlighted code snippets.

### Describe the solution you'd like
In addition to `[java]`, a `[kotlin]` block (and similar) would be helpful

**Comment by krzema12** [comment]
Hi @daincredibleholg, in my 2022 article for Java Advent I used embedded GitHub Gists snippets, see https://www.google.com/amp/s/www.javaadvent.com/2022/12/how-i-got-rid-of-yaml-in-githubs-ci.html/amp. The only disadvantage I see is that the article isn't self-contained, so if someone removes the gist, the article will lose the piece of code as well. Anyway, I think it's a good workaround at least.

**Comment by daincredibleholg** [comment]
You are absolutely correct, @krzema12. For now, it should do. We need a more portable / independent fix, though.
